vendor/github.com/openshift/source-to-image/pkg/build/strategies/sti/sti.go

```diff
@@ -562,7 +562,6 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 		}()
 
 		opts.Stdin = r
-		defer wg.Wait()
```

Comment:
I agree with @liggitt :)

@gabemontero we can write:

```go
go func() {                   // line 544
	defer wg.Done()
	// ...
```

... replacing lines [549](https://github.com/openshift/origin/pull/10776/files#diff-6f7433170556e6fb0dd5276a9f69e422R549) and [559](https://github.com/openshift/origin/pull/10776/files#diff-6f7433170556e6fb0dd5276a9f69e422R559).

Note `wg.Done()` != `wg.Wait()` (the code in this line we're commenting on).